ietf/static/js/ietf.js

```diff
@@ -244,15 +246,17 @@ $(function () {
                 extraNav.remove();
             }
 
-            $(window)
-                .on("scroll", function () {
+            $(document)
+                // Chrome apparently wants this debounced to something >10ms,
+                // otherwise the main view doesn't scroll?
+                .on("scroll", debounce(function () {
                     const item = $('#righthand-nav')
                         .find(".active")
                         .last();
                     if (item.length) {
                         item[0].scrollIntoView({ block: "center" });
                     }
-                });
+                }, 20));
```

Comment:
I would suggest using a value like `100` ms debounce if it achieves the same result. I've found using values under 100 ms to be unreliable in many scenarios.